example/package.json

```diff
@@ -3,7 +3,7 @@
   "name": "example",
   "version": "0.0.1",
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "start": "node ../node_modules/react-native/local-cli/cli.js start",
```

Comment:
Was it broken?

Reply:
Yes, I was not able to run a packager :(